falcon-tracer/conf/logging.ini

```diff
@@ -2,21 +2,27 @@
 keys=root
 
 [handlers]
-keys=consoleHandler
+keys=consoleHandler,fileHandler
 
 [formatters]
 keys=simpleFormatter
 
 [logger_root]
 level=DEBUG
-handlers=consoleHandler
+handlers=consoleHandler,fileHandler
 
 [handler_consoleHandler]
 class=StreamHandler
 level=DEBUG
```

Comment:
We can remove this and fallback to the default level, i.e., specified in the root logger.